src/layouts/Home/components/Search.tsx

```diff
@@ -35,8 +35,8 @@ export const Search: React.FC = () => {
   const { bannerImage } = useStaticQuery(
     graphql`
       {
-        bannerImage: abstractAsset(layerName: { eq: "home-hero-logo" }) {
-          childFile {
+        bannerImage: figmaAsset(name: { eq: "home-hero-logo" }) {
+          remoteImage {
```

Comment:
Is this API variation desirable? Seems like it will be an unwanted, less explicit `image` -> `children` impact for the rest of the codebase.